cmd/juju/crossmodel/list.go

```diff
@@ -34,20 +34,15 @@ options:
        - charm
 
 Examples:
-    $ juju list-offers
-    $ juju list-offers vendor:
-    $ juju list-offers --url vendor:/u/ibm
-    $ juju list-offers --interface db2
+    $ juju offers
+    $ juju offers user/model
+    $ juju offers --url user/model
```

Comment:
--interface is not yet supported so I wanted to remove the incorrect help text for now